front/app/api/reports/useAddReport.ts

```diff
@@ -8,13 +8,13 @@ import fetcher from 'utils/cl-react-query/fetcher';
 import reportsKeys from './keys';
 import { ReportResponse } from './types';
 
-type AddReport =
+export type AddReport =
   | {
       name: string;
-      phase_id?: never;
+      phase_id?: null;
     }
   | {
-      name?: never;
+      name?: null;
       phase_id: string;
```

Comment:
I think null/undefined makes more sense to me here rather than `never` :thinking: 

This change also allows me to easily type + generate a specific AddReport requestBody for the Community Monitor report vs others.

Let me know if you think I've missed an important reason for having `never` here though.